### ISSUE TITLE
htmlSafe imported incorrectly

### ISSUE THREAD

**Issue by BryanCrotaz** [Addepar/ember-table#972]
in `components/ember-table/component.js`
```js
import { htmlSafe } from '@ember/string';
```
should be
```js
import { htmlSafe } from '@ember/template';
```

**Comment by ascudder** [comment]
I opened this PR to fix this: https://github.com/Addepar/ember-table/pull/992

**Comment by ascudder** [comment]
@BryanCrotaz my PR with the fix was accepted and this should be fixed in version 5.0.5 of this addon!

**Comment by kpfefferle** [comment]
Resolved by #992
